Models/SupportedLanguages.ts

```diff
@@ -0,0 +1 @@
+﻿export type SupportedLanguages = 'javascript' | 'typescript' | 'unset';
```

Comment:
are we planning to actually support typescript?

Reply:
Ahh sorry I think that is left in from when I was debugging